CODEOWNERS

```diff
@@ -1,4 +1,4 @@
-*                                                                @costinm @linsun @howardjohn @stevenctl @ericvn @hzxuzhonghu
+*                                                                @costinm @linsun @howardjohn @stevenctl @keithmattix @hzxuzhonghu
```

Comment:
I think it's fine to add TOC group here.

Reply:
TOC is not targeted to review code, they are for arch design